docs/source/documentation/client_configuration.rst

```diff
@@ -115,3 +115,12 @@ such as passwords or tokens in environment variables.
     )
 
 
+Generating new API tokens
```

Comment:
I am not sure about putting this piece of information on this page (and then renaming something more general "How to login"). Or using a second page "How to deal with API token"

Reply:
I slightly favor the second option (new page) as it make each page clearer.
What do you think @IC-1101asterisk @SdgJlbl?

Reply:
I agree, I don't think this has anything to do with the rest of the page, it should be put somewhere else
Just waiting for @SdgJlbl to voice her opinion on this

Reply:
Yes, I think it's better to keep how-to guides short and focused on only one thing at a time :) 
You can create a new page "How-to generate new API tokens for login"